x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/sample_response/sample_response_logic.ts

```diff
@@ -68,12 +68,16 @@ export const SampleResponseLogic = kea<MakeLogicType<SampleResponseValues, Sampl
       const { http } = HttpLogic.values;
       const { engineName } = EngineLogic.values;
 
-      const url = `/api/app_search/engines/${engineName}/sample_response_search`;
+      const url = `/api/app_search/engines/${engineName}/search`;
 
       try {
         const response = await http.post(url, {
+          query: { query },
           body: JSON.stringify({
-            query,
+            page: {
```

Comment:
This was previously limited on the backend endpoint. Since a number of different UIs now use the same endpoint, I am doing the limiting on the front end now.